build/shade/EnvironmentParameters.shade

```diff
@@ -0,0 +1,20 @@
+use namespace='System'
+ 
+functions @{
+    static string GetEnvironmentParameter(string name)
+    {
+        return GetEnvironmentParameter(name, s => s);
+    } 
+
+    static T GetEnvironmentParameter<T>(string name, Func<string, T> valueFn)
+    {
+        var value = valueFn(Environment.GetEnvironmentVariable(name));
+        Console.WriteLine(name + ": " + (value != null ? value.ToString() : "(null)"));
```

Comment:
I think it depends on what the user does inside the Func because this is where the null will be passed. The only thing is they need to be aware of this possibility. And obviously the message for value types will never hit the path where the `(null)` is printed...